qiime_studio/server.py

```diff
@@ -13,6 +13,14 @@
 studio.after_request(add_cors_headers)
 
 
+# No output is visible, as for some reason it is held back
+# by the server flushed at seemingly random intervals
+# also, what should be the stdout, is being sent through stderr??
+def flush_stdio():
+    sys.stdout.flush()
+    sys.stderr.flush()
+
```

Comment:
Looking into it further we have a couple options, we can either monkey-patch `stdout`/`stderr` with writables which always flush the real file, OR there is just a [`flush`](https://docs.python.org/3/library/functions.html?highlight=print#print) keyword on print now.